bentoml/_internal/models/model.py

```diff
@@ -352,6 +392,36 @@ def from_dict(data: dict[str, str | dict[str, str]] | ModelContext) -> ModelCont
 
 @attr.frozen
 class ModelSignature:
+    """
+    A model signature represents a method on a model object that can be called.
+
+    Note that anywhere a ``ModelSignature`` is used, a ``dict`` with keys corresponding to the fields
+    can be used instead. For example, instead of ``{"predict": ModelSignature(batchable=True)}``, one
+    can pass ``{"predict": {"batchable": True}}``.
+
+    Fields:
+        batchable:
+            Whether the method should support batching. Note that batching must also
+            be explicitly enabled at runtime in the ``Model.to_runner`` or ``bentoml.runner`` call.
+        batch_dim:
+            The dimension(s) that contain multiple data. For example, if when passing two input arrays
+            ``[1, 2]`` and ``[5, 6]``, if the input array is ``[[1, 2], [3, 4]]``, then the batch dimension
```

Comment:
```suggestion
            ``[1, 2]`` and ``[3, 4]``, if the input array is ``[[1, 2], [3, 4]]``, then the batch dimension
```

Reply:
oh since we are recommending the `batch-to-batch` style API now, this may not be a great example. updated it slightly in the other comment.

Reply:
I think it's more explainable if you just use two arrays that you want to do prediction on and we talk about the input to the call, but I should make more clear what's going on, I'll rewrite this.